docs/content/en/docs/observability/prometheus.md

```diff
@@ -0,0 +1,144 @@
+```
+
+This will install Prometheus and Grafana in the `monitoring` namespace. Along with the Prometheus server, it'll also install other components viz. `node-exporter`, `kube-state-metrics` and `pushgateway`
+
+
+## Adding ServiceMonitor for Fission
+
+By default, this installation will not discover the metrics exposed by Fission. To be able to scrape those metrics, we can install ServiceMonitor, which is a Kubernetes  custom resource used to by the Prometheus Operator to add new targets.
+
+Create the manifest:
+
+```
+$ cat <<EOF > servicemonitors.yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: executor-service-app
+spec:
+  selector:
+    matchLabels:
+      svc: executor
+  endpoints:
+  - targetPort: 8080
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: router-service-app
+spec:
+  selector:
+    matchLabels:
+      svc: router
+  endpoints:
+  - targetPort: 8080
+EOF
+```
+
+Install the ServiceMonitors.
+
+```
+$ kubectl apply -f servicemonitors.yaml --namespace fission
+```
+
+This will install the ServiceMonitors in the `fission` namespace, the same where Fission is installed.
+
+## Accessing Grafana UI
+
+The installation creates a Service named `fission-metrics-grafana`. To access this, you can use Kubernetes port forwarding
+
+```
+$ kubectl --namespace monitoring port-forward svc/fission-metrics-grafana 3000:80
+```
+
+The Grafana can be now accessed on http://localhost:3000
+  
+This installation also adds Prometheus as a data source for Grafana automatically.
+You can verify and update this in the `Data Sources` section of the UI.
+
+# Metrics Queries
+
+Once Prometheus is configured, we can now run queries in Grafana over Fission metrics. Individual queries can be run under `Explore` section.
+
+Fission exposes a set of metrics. For example to query the total number of function calls, run
+
+```
+fission_function_calls_total
+```
+
+Calls for a specific function can be queried using
+```
+fission_function_calls_total{name="foo"}
+```
+
+To track the duration of a specific function
+```
+fission_function_duration_seconds{name="hello"}
+```
+
+There are other more Fission metrics, which can be found under `Metrics` in the Explore screen.
+
+# Fission Dashboard
+
+With Grafana, visuals dashboards can be created to monitor multiple metrics in an organized way.
+
+One such dashboard can be found [here](https://github.com/fission/examples/blob/master/dashboards/prometheus-fission-functions.json). This dashboard shows log metrics from all the major components of Fission.
+
+Once imported, the dashboard will look similar to below image.
+
+{{< img "../assets/prometheus-grafana.png" "Prometheus Fission Functions dashboard" "30em" "1" >}}
```

Comment:
The image is quite small - possible to have slightly bigger if not too much effort?

Reply:
Changed the image markdown to responsive, relative to the sizeof other content